core/commands/name/name.go

```diff
@@ -84,27 +79,25 @@ Resolve the value of a dnslink:
 }
 
 type IpnsInspectValidation struct {
-	Valid     bool
-	Reason    string
-	PublicKey peer.ID
+	Valid  bool
+	Reason string
+	Name   string
 }
 
 // IpnsInspectEntry contains the deserialized values from an IPNS Entry:
 // https://github.com/ipfs/specs/blob/main/ipns/IPNS.md#record-serialization-format
 type IpnsInspectEntry struct {
-	Value        string
-	ValidityType *ipns_pb.IpnsEntry_ValidityType
+	Value        *path.Path
+	ValidityType *ipns.ValidityType
 	Validity     *time.Time
-	Sequence     uint64
-	TTL          *uint64
-	PublicKey    string
-	SignatureV1  string
-	SignatureV2  string
-	Data         interface{}
+	Sequence     *uint64
+	TTL          *time.Duration
 }
 
 type IpnsInspectResult struct {
 	Entry      IpnsInspectEntry
```

Comment:
Can we add protobuf size and print it before the hexdump and Version?
Will help with quick eyeballing and other analysis (V1 vs V2 vs records with additional data in Data CBOR)

```suggestion
	Entry      IpnsInspectEntry
	PbSize       int
```